WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -178,10 +178,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 {
     DDLogVerbose(@"didFinishLaunchingWithOptions state: %d", application.applicationState);
 
-    // Launched by tapping a notification
-    if (application.applicationState == UIApplicationStateActive) {
```

Comment:
I actually removed this because the comment was not correct from what I saw/read. I will double-check however.